airbyte-integrations/connectors/source-simplesat/manifest.yaml

```diff
@@ -0,0 +1,407 @@
+version: 5.8.1
+
+type: DeclarativeSource
+
+description: |
+  The Simplesat source connector which ingests data from the Simplesat API.
+  Simplesat is a engaging survey tool for service businesses to gather insight and feedback from their customers.
+
+  The source supports 3 of the endpoints from the API (questions, surveys and answers)
+  For more information, checkout the website https://www.simplesat.io/
+  
+  An API key is required for authentication and using this connector.
+  In order to obtain an API key, you must first create a Simplesat account.
+  Once logged-in, you will find your API key in the account settings.
+  You can find more about their API here https://documenter.getpostman.com/view/457268/SVfRt7WJ?version=latest
+  
+check:
+  type: CheckStream
+  stream_names:
+    - surveys
+
+definitions:
+  streams:
+    surveys:
+      type: DeclarativeStream
+      name: surveys
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /surveys
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - surveys
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            inject_into: request_parameter
+            field_name: page
+          page_size_option:
+            type: RequestOption
+            field_name: page_size
+            inject_into: request_parameter
+          pagination_strategy:
+            type: PageIncrement
+            page_size: 100
+            start_from_page: 1
+            inject_on_first_request: true
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/surveys"
+    questions:
+      type: DeclarativeStream
+      name: questions
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: questions
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - questions
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            inject_into: request_parameter
+            field_name: page
+          page_size_option:
+            type: RequestOption
+            field_name: page_size
+            inject_into: request_parameter
+          pagination_strategy:
+            type: PageIncrement
+            page_size: 100
+            start_from_page: 1
+            inject_on_first_request: true
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/questions"
+    search:
```

Comment:
This endpoint must be called answers not search.

Reply:
Also you can implement `responses` stream using the same logic.

Reply:
Implemented